src/playground/Widgets/List/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { ProgressBar } from "@/components/Charts/ProgressChart"
+
+interface BaseListItem {
+  name: string
+}
+
+interface DefaultListItem extends BaseListItem {
+  value: number | string
+}
+
+interface ProgressListItem extends BaseListItem {
+  value: number
+}
+
+type ListItem = DefaultListItem | ProgressListItem
+
+interface ListProps {
+  items: ListItem[]
+  title?: string
+  variant?: "default" | "progress"
+}
```

Comment:
We could improve this type to better restrict it, so that we don't allow anyone to pass both progress items and text items

```suggestion
type ListProps = {
  title?: string
} & ({
  items: DefaultListItem[]
} | {
  items: ProgressListItem[]
})
```

We won't need the `variant` either

Reply:
Cool!